src/CrossOrigin.ts

```diff
@@ -401,7 +401,7 @@ interface CrossOriginElementDataIn {
     /**
      * Optionally wait if the element is accessible or focusable before returning it
      */
-    accessibility?: Types.ObservedElementAccesibility;
+    accessibility?: Types.ObservedElementAccessibility;
```

Comment:
FYI - the rename is a breaking change, please keep that in mind when bumping the next release

Reply:
Will do that adjustment as a separate PR.